core/replacement_test.go

```diff
@@ -126,16 +127,6 @@ level1:
 	then.FileContents(t, endData, filepath)
 }
 
-func TestErrorBadFileRead(t *testing.T) {
```

Comment:
I removed this since the call to `filepath.Glob` will return all existing files/directories that match the given pattern.

Reply:
Hmm, I do think this test is actually valid as we would still want to return an error if the glob found no files. So maybe not checking for error but `len(files) == 0`